### PR TITLE
Ds001 populate db fix after testing

### PR DESCRIPTION
**Pull request by MartaMartaZielonka**

### CODE REVIEW

File: src/databaseScripts/CreateTables.sql [hunk 1]
```diff
@@ -53,7 +53,8 @@ CREATE TABLE IF NOT EXISTS training_employee(
 		REFERENCES employee(id)
 );
 
-CREATE TABLE IF NOT EXISTS employees_fact(
+
```
Comment (EwaRamlo) [inline src/databaseScripts/CreateTables.sql]: remove blank line

File: src/databaseScripts/InsertData.sql [hunk 2]
```diff
@@ -16,7 +16,8 @@ INSERT INTO band(band_level, band_name)
 	(6, "Apprentice"),
 	(6, "Trainee");
 
-INSERT INTO trainings(training_name, is_obligatory)
+
```
Comment (EwaRamlo) [inline src/databaseScripts/InsertData.sql]: remove blank line

File: src/databaseScripts/InsertData.sql [hunk 3]
```diff
@@ -37,4 +38,3 @@ INSERT INTO trainings(training_name, is_obligatory)
 	("Wellbeing Guides", 0),
 	("Unconscious Bias", 0),
 	("Workday - Bitesize 101", 0);
-
```
Comment (EwaRamlo) [inline src/databaseScripts/InsertData.sql:40]: add newline at the end of file

**Review by EwaRamlo — CHANGES_REQUESTED** [review]

File: src/databaseScripts/CreateTables.sql (hunk 1 above)
Comment (MartaMartaZielonka) [reply src/databaseScripts/CreateTables.sql]: done

File: src/databaseScripts/InsertData.sql (hunk 2 above)
Comment (MartaMartaZielonka) [reply src/databaseScripts/InsertData.sql]: done

File: src/databaseScripts/InsertData.sql (hunk 3 above)
Comment (MartaMartaZielonka) [reply src/databaseScripts/InsertData.sql:40]: line added

**Review by EwaRamlo — APPROVED** [review]

**Review by shaunganley — APPROVED** [review]

**Review by JohnWalker-Kainos — APPROVED** [review]

**Pull request merged**